e2e/tests/conceptsets/copy-to-workspace.spec.ts

```diff
@@ -18,41 +40,26 @@ describe('Copy Concept Set to another workspace', () => {
 
   /**
    * Test:
-   * - Copy Concept Set from one workspace to another workspace.
+   * - Copy Concept Set from one workspace to another workspace when both have the same CDR Version.
    */
-  test('Workspace OWNER can copy Concept Set', async () => {
+  test('Workspace OWNER can copy Concept Set when CDR Versions match', async () => {
 
-    // Need a source and a destination workspace.
+    // Create a source and a destination workspace with the same CDR Version.
 
-    const destWorkspaceCard: WorkspaceCard = await findWorkspace(page, {create: true});
```

Comment:
TODO: refactor the kitchen-sink `findWorkspace()`.  All we are using it for here is workspace creation.